common/djangoapps/course_modes/views.py

```diff
@@ -39,7 +38,7 @@ class ChooseModeView(View):
     """
 
     @method_decorator(transaction.non_atomic_requests)
-    def dispatch(self, *args, **kwargs):        # pylint: disable=missing-docstring
+    def dispatch(self, *args, **kwargs):  # pylint: disable=missing-docstring
```

Comment:
I'm not completely sure about all the use-cases for this view so I'm going to place a general description for this code.